### DISCUSSION TITLE
Search in scatter or sunburst.

### DISCUSSION THREAD

**Discussion by kramerrs** [plotly/plotly.R#2369] [Q&A]
I have a beautiful interactive sunburst up and running, with interactive events that can sync with other plots.

Since the plot is more than three levels deep and many of the nodes slices are too small to click on.  I would like to implement a search.  I am wondering if there is any way to go about programmatically zooming to a node?  Especially in R.  Ideally I would like to be able to highlight the node in the sunburst and in the scatter plot and a bar chart.

**Comment by romanzenka** [comment]
Not sure if the `crosstalk` package could help you somehow in this situation, but that would be the place I would look at first.
